src/Popover/helpers/getScrollableParent.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+import type { GetScrollableParent } from "./getScrollableParent";
+
+const regex = /(auto|scroll)/;
+
+const style = (node, prop) => getComputedStyle(node).getPropertyValue(prop);
+
+const scroll = node =>
+  regex.test(style(node, "overflow") + style(node, "overflow-y") + style(node, "overflow-x"));
```

Comment:
Is it checking just for `overflow` attributes enough? What if `overflow` will be present, but the actual `scrollingHeight` won't be bigger than `clientHeight`?

Reply:
Good point I will investigate that

Reply:
In that case, listener will be attached but won't callback anything, due to element not being scrollable, we can talk about if we want to attach listener to element which is not scrollable but in my mind is ok due to resize or dynamic content change

Reply:
IMHO, it's needed to add tests only for `getScrollableParent`. The other functions are not exported and they doesn't have to be - there is no other use case yet.
I would rather add comments to all functions that are declared there 🙂

Reply:
If we need also to test other cases - if `style` works alright, we can use these cases for `getScrollableParent`.